org/chapter3.org

```diff
@@ -95,7 +95,7 @@
 #+end_src
 
 செயல்பாடுகளை அவற்றின் பெயரை எழுதி, காற்புள்ளிகளால்(Comma - ,) பிரிக்கப்பட்ட
-அடைப்புக்குறிக்குள் வாதங்களை வைப்பதன் மூலம் செயல்பாடுகளை
+அடைப்புக்குறிக்குள் வாதங்களை(argumetents) வைப்பதன் மூலம் செயல்பாடுகளை
```

Comment:
Can you change arguments into தருமதிப்புகள். i have used this translation throughout this book but i missed here.

Reference: https://ta.wiktionary.org/wiki/arguments